bin.js

```diff
@@ -19,16 +19,13 @@ const parseJSON = input => {
 const joycon = new JoyCon({
   parseJSON,
   files: [
+    'pino-pretty.config.cjs',
```

Comment:
I would add also `mjs`.

Reply:
mjs files can only be loaded dynamicly using `import()`, which is asynchronous. Joycon is synchronous and uses `require()`. So that can't really work. At least not simply.